### DISCUSSION TITLE
建议添加F搜翻译

### DISCUSSION THREAD

**Discussion by codeleep** [YiiGuxing/TranslationPlugin#2451] [Ideas]
这个翻译免费，对新用户也是一个好的选择：https://fsoufsou.com/translate

**Comment by YiiGuxing** [comment]
感谢您的建议，这个看着也是不错的，不过还是要经过人家同意才行的（大概率是不会同意的），可以尝试沟通一下。不过现在计划任务排得太多了，也不知道什么时候才有时间处理。。。

**Reply by YiiGuxing** [reply]
> @YiiGuxing 你好! 我们是F搜的开发者, 很喜欢你们的作品, 我们也是用户之一! 只要在翻译结果界面标注 "Powered by F搜" 即可! 如果有具体问题, 可以联系我 wangconnor#gmail#com

@connorwang 很高兴听到你们的声音，感谢你们能同意接入F搜翻译，后面F搜翻译肯定会被接入的，再次感谢！

**Comment by L-Super** [comment]
有最新进展了吗
